eval/src/braintrust.ts

```diff
@@ -76,18 +77,42 @@ export async function runEval(args: Params, onSummary: OnSummaryFn) {
   // Change working directory
   process.chdir(path.resolve(root));
 
-  let command: string;
   const terminateFlag = terminate_on_failure ? "--terminate-on-failure" : "";
 
-  switch (args.runtime) {
-    case "node":
-      command = `npx braintrust eval --jsonl ${terminateFlag} ${paths}`;
-      break;
-    case "python":
-      command = `braintrust eval --jsonl ${terminateFlag} ${paths}`;
-      break;
-    default:
-      throw new Error(`Unsupported runtime: ${args.runtime}`);
-  }
+  const baseCommand = (() => {
+    switch (args.runtime.toLowerCase().trim()) {
+      case "node":
+        switch (args.package_manager) {
+          case "":
```

Comment:
Wondering if default should be defined and this should be removed -- what happens in this empty case?

Reply:
yeah for backward compatibility it'll be whatever we were doing before. so for node it'll be npx and for python it'll be pip. is that what you had in mind?

Reply:
yeet i forgot about the case statement behavior on no returns -- this makes sense